packages/design-system-react-native/src/components/Text/Text.stories.tsx

```diff
@@ -0,0 +1,157 @@
+import type { Meta, StoryObj } from '@storybook/react-native';
+import React from 'react';
+import { View } from 'react-native';
+
+import Text from './Text';
+import type { TextProps } from './Text.types';
+import { TextVariant, FontWeight, FontStyle, TextColor } from './Text.types';
+
+const meta: Meta<TextProps> = {
+  title: 'Components/Text',
+  component: Text,
+  args: {
+    variant: TextVariant.BodyMd,
+    color: TextColor.TextDefault,
+    fontWeight: FontWeight.Normal,
+    fontStyle: FontStyle.Normal,
+    children: 'The quick orange fox jumped over the lazy dog.',
+    twClassNames: '',
+  },
+  argTypes: {
+    variant: {
+      control: 'select',
+      options: Object.values(TextVariant),
+    },
+    color: {
+      control: 'select',
+      options: Object.values(TextColor),
+    },
+    fontWeight: {
+      control: 'select',
+      options: Object.values(FontWeight),
+    },
+    fontStyle: {
+      control: 'select',
+      options: Object.values(FontStyle),
+    },
+    children: {
+      control: 'text',
+    },
+    twClassNames: {
+      control: 'text',
+    },
+  },
+};
+
+export default meta;
+type Story = StoryObj<TextProps>;
+
+export const Default: Story = {
+  render: () => (
+    <Text variant={TextVariant.BodyMd}>
+      The quick orange fox jumped over the lazy dog.
+    </Text>
+  ),
+};
```

Comment:
I noticed that the Color story appears before Default is this intentional?

<img width="440" alt="Screenshot 2024-11-27 at 11 57 39 AM" src="https://github.com/user-attachments/assets/ee1028d5-cbf8-484e-bb02-2d54aa294527">